lazy-pages/src/lib.rs

```diff
@@ -254,6 +254,29 @@ pub fn get_released_pages() -> Vec<PageNumber> {
     LAZY_PAGES_CONTEXT.with(|ctx| ctx.borrow().released_pages.iter().copied().collect())
 }
 
+/// TODO: remove this after current test-net chain will be dropped (issue #1317).
+/// This patch solves the problem for block `#866245`, see more in issue.
+pub fn get_released_pages_patch() -> Vec<PageNumber> {
+    LAZY_PAGES_CONTEXT.with(|ctx| {
+        let ctx = ctx.borrow_mut();
+        if ctx.program_storage_prefix.as_ref().unwrap()
+            == &vec![
```

Comment:
Use `lazy_pages` crate:
```suggestion
            == &lazy_pages::pages_prefix(...)
```

Reply:
It has additional  4 bytes for page prefix in the end. I think better is to use clear prefix without additional, to avoid additional bytes in right vector.

Reply:
> It has additional 4 bytes for page prefix in the end.

It's `page_key`, not `pages_prefix`

Reply:
Okey, but what the difference ?)

Reply:
Oops. It's `common` crate where code is:
```rust
key.extend(STORAGE_PROGRAM_PAGES_PREFIX);
key.extend(program_id.as_fixed_bytes());
key.extend(b"::");
```

Reply:
Discussed tet a tet